src/app/modules/shared-components/components/input-error/input-error.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { AbstractControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'alg-input-error',
+  templateUrl: './input-error.component.html',
+  styleUrls: [ './input-error.component.scss' ],
+})
+export class InputErrorComponent implements OnChanges {
+  @Input() form?: FormGroup;
+  @Input() inputName = '';
+
+  control?: AbstractControl;
+
+  constructor() {
+  }
+
+  ngOnChanges(): void {
+    const elm = this.form?.get(this.inputName);
+    if (!elm) return;
+    this.control = elm;
```

Comment:
Ok so you can just write that I think.
```suggestion
    this.control = this.form?.get(this.inputName);
```

Reply:
I've done some changes to make the fact that elm can be null clearer